test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2215,10 +2213,9 @@ def test_cgroup_cpuset_memory_spread_page(self):
         self.server.status(JOB, ATTR_o, jid)
         o = j.attributes[ATTR_o]
         self.tempfile.append(o)
-        fn = self.get_cgroup_job_dir('cpuset', jid, self.hosts_list[0])
+        fn = self.get_cgroup_job_dir('cpuset', jid, hostname)
         fn = os.path.join(fn, spread_path)
-        result = self.du.cat(hostname=self.hosts_list[0],
-                             filename=fn, sudo=True)
+        result = self.du.cat(hostname, filename=fn, sudo=True)
```

Comment:
hostname=hostname
You might want to consider renaming your hostname variable to hn or hostn.